server/integration/endpoint/src/main/resources/schema/jboss-infinispan-endpoint_7_0.xsd

```diff
@@ -160,7 +160,7 @@
                     other nodes. Defaults to true.</xs:documentation>
             </xs:annotation>
         </xs:attribute>
-        <xs:attribute name="await-initial-transfer" type="xs:boolean" use="optional" default="true">
+        <xs:attribute name="await-initial-retrieval" type="xs:boolean" use="optional" default="true">
```

Comment:
Not changing the parser, just the schema so that it matches what the parser actually understands :)